src/concrete/ml/common/utils.py

```diff
@@ -34,10 +34,9 @@
 
 MAX_BITWIDTH_BACKWARD_COMPATIBLE = 8
 
-# Indicate if the old virtual library method should be used instead of the compiler simulation
-# when simulating FHE executions
-# Set 'USE_OLD_VL' to False by default once the new simulation is fixed
-# FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4091
+# Use new VL with .simulate() by default once CP's multi-parameter/precision bug is fixed
```

Comment:
is this change expected ? I would  revert it as the correct link to put here is https://github.com/zama-ai/concrete-ml-internal/issues/4091 and not https://github.com/zama-ai/concrete-ml-internal/issues/3856 !

Reply:
Really don't know what happened here